.github/workflows/rust.yml

```
@@ -30,9 +30,8 @@ jobs:
         env:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
 
-      - uses: Swatinem/rust-cache@v2
-
-      - run: rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
+      - name: Cache Cargo dependencies
+        uses: actions/cache@v4
```

actionlint (1.7.7)

missing input "key" which is required by action "actions/cache@v4". all required inputs are "key", "path"

(action)